### PR TITLE
SF-10096 - Rep onboarding static screens

### PR DESCRIPTION
**Pull request by Mikechaos**

### CODE REVIEW

**Comment by Mikechaos** [comment]
Tiens un petit screenshot pour le fun
screen shot 2015-11-25 at 11 44 42 pm

File: app/js/components/Input.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
+import React, {PropTypes} from 'react';
+import ReactDOM from 'react-dom';
+import classnames from 'classnames';
+import zxcvbn from 'zxcvbn';
+
+import LabelElement from 'app/js/components/forms/LabelElement';
+
+window.zxcvbn = zxcvbn;
```
Comment (posabsolute) [inline app/js/components/Input.js]: We are going to move that into the password strength component I guess?

File: app/js/components/Input.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+
+class InputElement extends React.Component {
+  componentWillMount() {
+    const currentBlur = this.props.field.onBlur;
+    const currentChange = this.props.field.onChange;
+    this.props.field.onBlur = this.onBlurProxy.bind(this, currentBlur);
```
Comment (posabsolute) [inline app/js/components/Input.js]: Why do we need to proxy?

File: app/js/components/Input.js [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+      </div>
+    );
+  }
+
+  getClasses() {
+    return classnames({'salesfloor-form__label': true, [this.props.icon]: true});
+  }
+}
+
+class PasswordValidation extends React.Component {
```
Comment (posabsolute) [inline app/js/components/Input.js]: let's divide into multiple component files? Makes for a long file

File: app/js/components/forms/pages/EnterToken.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+class EnterTokenForm extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    console.log('EnterTokenForm');
+  }
+
+  getProps() {
```
Comment (posabsolute) [inline app/js/components/forms/pages/EnterToken.js]: Kinda duplicate code with line 23

File: app/js/components/forms/pages/PickStore.js [hunk 5]
```diff
@@ -0,0 +1,96 @@
+
```
Comment (posabsolute) [inline app/js/components/forms/pages/PickStore.js]: Just wondering, I saw a lot of repos using containers  folder as a practice, should we consider pages containers?

Comment (Mikechaos) [inline app/js/components/forms/pages/PickStore.js]: Container, as I see them, are components that use react-redux `connect()` function and are aware of redux.
Is that what you mean?

**Pull request merged**